### DISCUSSION TITLE
Excuse me, when calling the renderScore method in the ios real browser, a rangeerror: Maximum call stack size exceeded appears. Is it because there is a size limit?

### DISCUSSION THREAD

**Discussion by hhly** [CoderLine/alphaTab#1176] [Q&A]
<img width="451" alt="image" src="https://github.com/CoderLine/alphaTab/assets/29226405/335cd79a-e61e-4ac8-bc27-95a18a094e58">Excuse me, when calling the renderScore method in the ios real browser, a rangeerror: Maximum call stack size exceeded appears. Is it because there is a size limit?

**Comment by Danielku15** [comment]
From the file name it looks like you are bundling alphaTab (WebPack, Rollup or something else?). I would guess the problem resides within your application, not alphaTab in general. But if you can provide some sample project I can maybe see something. If you search in our discussions, issues and documentation for WebPack and Bundler related topics you will also find some hints around that.
